### ISSUE TITLE
Implement arbitrum.zerodao.com workflow

### ISSUE THREAD

**Issue by raymondpulver** [zerodao-finance/zeroBRIDGE#2]
Implement the workflow as follows:

User enters in the BTC he wants to move to Arbitrum, it will show him the amount of renBTC and arbETH he will get in the two button labels.

Slider will be set to show the allocation of arbETH user is getting out of the swap, out of the full amount for 100% conversion

Should have some sane default for an acceptable amount of arbETH to work with

For default ETH allocation, check arbiscan.io for the cost of an average curve.fi transaction and multiply that by 50. In React code, take the maximum of that constant compared to 10% of the swap

The minimum amount of arbETH allocation should be 3x the cost of a renBTC release from Arbitrum back to BTC. Refer to arbiscan.io to ascertain a good constant.

When the user sees the amounts he will get and has specified the address he wants to receive the funds on Arbitrum, he should press swap and it should prompt him to sign using (await  transferRequest.sign(signer))

UI should transition to a frame that should cache all parameters using the LocalStoragePersistenceAdapter class that is provided to the TransferRequest object, and there should be a way in the React code where the user can resume the state of the workflow if there is an issue, and he's already sent the BTC. The method that bridge.renproject.io uses to implement this is the URL of the page itself, which can be bookmarked. Something like this is acceptable. Our LocalStoragePersistenceAdapter will store all TransferRequest objects in localStorage to be retrieved. It may make sense to inherit from this PersistenceAdapter class to make something that works well for this application.

It should present the BTC address on the frame following signature, and it should show the progress over the following events:

1) user deposit is detected (await transferRequest.pollForFromChainTx()) 
2) renBTC is loaned and held in escrow on the contract (use a log filter on the Ethereum events that are relevant) 
3) tx goes from 0 / 6 confirmations on BTC chain  (check code for BTCBackend where you can get the confirmation count for a deposit address)
4) funds are released to destination (query the module contract for the swap indexed by the pNonce you are watching)

When it's done it should transition to a final frame that just shows a "Done!" button that takes the user back to the main screen

To summarize, all the user has to do for the application is click his intent to mint renBTC/arbETH at the ratio he wants, sign the message, then use the BTC address he is given to deposit, then wait.
